src/pages/edit_resource/EditResourcePage.tsx

```diff
@@ -257,18 +260,23 @@ const EditResourcePage: FC = () => {
   };
```

Comment:
Siden generelle kommentarer ikke kunne resolves så får jeg skrive de inn her:
I AppRoutes burde ikke pathen til å redigere eksisterende ressurs vært /editresource/{id} eller noe lignende istedet for /registration/{id}?

Reply:
lurte litt på det jeg også - sier meg enig og fikser det :-)